packages/css-library/dist/stylesheets/utilities.css

```diff
@@ -6247,16 +6247,16 @@ through all possible variants
 }
 
 .vads-u-margin-x--auto {
-  margin-left: -1px !important;
-  margin-right: -1px !important;
+  margin-left: auto !important;
+  margin-right: auto !important;
 }
```

Comment:
I just want to add a reminder for us to update the [margin utility docs](https://design.va.gov/foundation/utilities/margins) so that it's displaying the correct styles for these classes (instead of `-1px`):

- vads-u-margin-x--auto
- vads-u-margin-right--auto
- vads-u-margin-left--auto